templates/origintrials/ot-beta-availability-email.html

```diff
@@ -0,0 +1,10 @@
+According to go/chrome-schedule, Chrome {{release_milestone}} will soon arrive on the
+beta channel.<br /><br />
+Check that your feature shows up correctly under the origin trials section of
```

Comment:
Added the name in parentheses to clarify. This section is pointing the user to make sure it shows up in the roadmap,  and a link exists here for that, so that seemed sufficient to me for this email's use case. 🙂